Sources/SwiftFormatConfiguration/Configuration.swift

```diff
@@ -120,6 +120,12 @@ public struct Configuration: Codable, Equatable {
     self.version = highestSupportedConfigurationVersion
   }
 
+  /// Constructs a Configuration using values from a JSON config file
```

Comment:
Let's leave the format of the file unspecified in the doc comment, so that we have the ability to change it in the future (while still retaining support for older formats):

```
/// Constructs a Configuration by loading it from a configuration file.
```